packages/tasks/src/local-apps.ts

```diff
@@ -262,6 +262,43 @@ const snippetTgi = (model: ModelData): LocalAppSnippet[] => {
 	];
 };
 
+const snippetMlxLm = (model: ModelData): LocalAppSnippet[] => {
+	const openaiCurl = [
+		"# Calling the OpenAI-compatible server with curl",
+		`curl -X POST "http://localhost:8000/v1/chat/completions" \\`,
+		`   -H "Content-Type: application/json" \\`,
+		`   --data '{`,
+		`     "model": "${model.id}",`,
+		`     "messages": [`,
+		`       {"role": "user", "content": "Hello"}`,
+		`     ]`,
+		`   }'`,
+	];
+
+	return [
+		{
+			title: "Generate or start a chat session",
+			setup: ["# Install MLX LM", "pip install mlx-lm"].join("\n"),
+			content: [
+				"# One-shot generation",
+				`mlx_lm.generate --model "${model.id}" --prompt "Hello"`,
```

Comment:
no need for this one in case the model is conversational, IMO

Reply:
it still work and can be useful but I removed it (it's better for the copy button to keep one snippet only)